### PR TITLE
ADEN-2815 Evolve2 section param

### PR DESCRIPTION
**Pull request by fraszczakszymon**
Please do CR for single commits:
- fixed calculation of section param for Evolve2 provider (now it's based on `s0` and `s0v` params)
- changed `getMappedVertical` method name to `getName` because it's more connected to dbname

### CODE REVIEW

File: extensions/wikia/AdEngine/js/spec/provider/evolve2.spec.js [hunk 1]
```diff
@@ -1,33 +1,41 @@
 /*global beforeEach, describe, it, modules, expect, spyOn*/
-describe('OpenX Provider targeting', function () {
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine/js/spec/provider/evolve2.spec.js:2]: Good eye :bird:

File: extensions/wikia/AdEngine/js/provider/evolve2.js [hunk 2]
```diff
@@ -17,7 +17,6 @@ define('ext.wikia.adEngine.provider.evolve2', [
 			'320x50,320x100,300x250': 'a',
 			'300x250': 'a'
 		},
-		section = evolveHelper.getSect(),
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine/js/provider/evolve2.js:20]: It seems we don't use `getSect()` anylonger. Shouldn't we remove it from `evolveHelper`?

**Comment by nandy-andy** [comment]
LGTM :sunglasses:

File: extensions/wikia/AdEngine/js/provider/evolve2.js (hunk 2 above)
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/provider/evolve2.js:20]: It is used by previous evolve provider:
https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/provider/evolve.js#L32

We'll be able to remove whole evolveHelper once we decide to move on Evolve2

Comment (nandy-andy) [inline extensions/wikia/AdEngine/js/provider/evolve2.js:20]: Oh, OK. I might've done something wrong while grepping I didn't spot that. Thanks for update!

**Pull request merged**